change/@fluentui-react-components-b71ce9f6-1c6d-406f-b8ef-192483756f06.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
This is `minor` in `master` but I've changed it to patch here so it can be picked up in `9.0.2` at @layershifter's request.